vuu-ui/packages/vuu-filters/src/filter-clause/ExpandoCombobox.tsx

```diff
@@ -21,6 +21,7 @@ const classBase = "vuuExpandoCombobox";
 export interface ExpandoComboboxProps<Item = string>
```

Comment:
There are no cypress tests for ExpandoComboBox because its such a thin layer round combobox (not that its a good reason there should really be some)
There are some showcase examples, though. IT would be good to have one example that shows the new prop in use